.github/workflows/lint.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Lint Code Base
-        uses: docker://github/super-linter:v2.1.1
+        uses: github/super-linter@v4
```

Comment:
We could alternatively just remove this workflow. It appears to be ignoring Pull Requests targeting master, so it almost never runs.

Reply:
Having the shellcheck run might be of some value, so what if we update the trigger? Maybe something like this?
```
on:
  push:
    branches-ignore: [master]
  pull_request:
    branches: [master]
```